libs/ngx-charts-on-fhir/src/lib/fhir-chart-layout/fhir-chart-layout.component.ts

```diff
@@ -1,17 +1,43 @@
 import { ChangeDetectionStrategy, Component, Input, OnChanges } from '@angular/core';
-import { MatSidenav } from '@angular/material/sidenav';
-import { ToolbarButtonName } from '../data-layer-toolbar/data-layer-toolbar/data-layer-toolbar.component';
+import { MatSidenav, MatSidenavModule } from '@angular/material/sidenav';
+import { DataLayerToolbarComponent, ToolbarButtonName } from '../data-layer-toolbar/data-layer-toolbar/data-layer-toolbar.component';
 import { SharedDataLayerListService } from '../data-layer-list/shared-data-layer-list.service';
 import { FhirDataService } from '../fhir-data/fhir-data.service';
+import { CommonModule } from '@angular/common';
+import { MatIconModule } from '@angular/material/icon';
+import { PatientBrowserComponent } from '../patient-browser/patient-browser.component';
+import { DataLayerListComponent } from '../data-layer-list/data-layer-list/data-layer-list.component';
+import { DataLayerBrowserComponent } from '../data-layer-browser/data-layer-browser.component';
+import { PatientService } from '../patient-browser/patient.service';
 
 /**
  * See `*ChartLayout` for example usage.
  */
 @Component({
+  standalone: true,
+  imports: [
+    CommonModule,
+    MatSidenavModule,
+    MatIconModule,
+    DataLayerToolbarComponent,
+    PatientBrowserComponent,
+    DataLayerListComponent,
+    DataLayerBrowserComponent,
+  ],
   selector: 'fhir-chart-layout',
   templateUrl: './fhir-chart-layout.component.html',
   styleUrls: ['./fhir-chart-layout.component.css'],
   changeDetection: ChangeDetectionStrategy.OnPush,
+  providers: [
+    {
+      provide: SharedDataLayerListService,
+      useClass: SharedDataLayerListService,
+    },
+    {
+      provide: PatientService,
+      useClass: PatientService,
+    },
```

Comment:
`PatientService` is provided in both `FhirChartLayoutComponent` and showcase `AppComponent`, so they are using different instances of the service and the app will not update when you select a patient in the patient browser.